controllers/nginx/configuration.md

```diff
@@ -110,49 +109,47 @@ Indicates the [HTTP Authentication Type: Basic or Digest Access Authentication](
 ingress.kubernetes.io/auth-secret:secretName
 ```
 
-Name of the secret that contains the usernames and passwords with access to the `path/s` defined in the Ingress Rule.
-The secret must be created in the same namespace than the Ingress rule
+The name of the secret that contains the usernames and passwords with access to the `path`'s defined in the Ingress Rule.
+The secret must be created in the same namespace than the Ingress rule.
```

Comment:
Is this meant to say "as the" instead of  "than the" ? Or is this implying ordering?